project/SbtMultiJvm.scala

```diff
@@ -166,6 +170,43 @@ object MultiJvmPlugin extends AutoPlugin {
         name.value + "_" + scalaVersion.value + "-" + version.value + "-multi-jvm-assembly.jar"
       })
 
+  def collectMultiJvmTests(
```

Comment:
This replicates some of sbt internal logic:

https://github.com/sbt/sbt/blob/a199875e1dcb16cd92a439b47379954daff1a995/main-actions/src/main/scala/sbt/Tests.scala#L251-L300

But tailored to our use case. Later when we move this back to sbt-multi-jvm plugin, we can see how to modernize the plugin according to sbt's new apis.